worlds/oot_soh/Enums.py

```diff
@@ -3463,6 +3513,7 @@ class Events(str, Enum):
     DRAIN_WELL = "Drain Well"
     GAME_COMPLETED = "Game Completed"
     CAN_ACCESS_BLUE_FIRE = "Can Access Blue Fire"
+    BORROW_BUNNY_HOOD = "Borrow Bunny Hoodd"
```

Comment:
I've added an event on my end for CAN_BORROW_BUNNY_HOOD instead, please use that one instead and remove this one.